src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -243,7 +243,7 @@ class SharedLinkSection extends React.Component<Props, State> {
                             buttonProps={copyButtonProps}
                             className="shared-link-field-container"
                             disabled={submitting}
-                            label=""
+                            label={intl.formatMessage(messages.sharedLinkURLLabel)}
```

Comment:
What's the visual change here? It has an aria label, icon, and tooltip. Is this adding a text label?

Reply:
@alexkrolick @sahiga I'll close this PR because this label should be empty.